dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -73,8 +70,17 @@ def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions,skipClear = False
             for row in reader:
                 skipInsert = False
                 for field in fieldOptions:
+                    if row[field] is None:
+                        # If field is empty set to an empty string
+                        row[field] = ""
                     # For each field with options present, modify according to those options
                     options = fieldOptions[field]
+                    if "strip_commas" in options:
+                        # Remove commas from numeric fields
+                        row[field] = row[field].replace(",","")
+                        if row[field] == "":
```

Comment:
If this causes an error in calculations, it will prompt the agency to update their data with GTAS/SAM/whatever it's called, which is kind of the point.